xprof/xprof.rb.in

```diff
@@ -618,6 +618,11 @@ def enable_events_metadata(channel_name, tracing_mode: 'default', profiling: tru
   exec("#{lttng_enable} lttng_ust_thapi:*")
 end
 
+def enable_events_profiler(channel_name, tracing_mode: 'default', profiling: true)
```

Comment:
I think I prefer `toggle` :) sorry!

Reply:
No, I was also going to use `toggle` -- right now it is not consistent and switch between `toggle` and `profiler` in various places.